5-network/01-fetch/01-fetch-users/_js.view/test.js

```diff
@@ -1,7 +1,7 @@
 describe("getUsers", function() {
 
-  it("gets users from GitHub", async function() {
-    let users = await getUsers(['iliakan', 'remy', 'no.such.users']);
+  it("pobiera użytkowników GitHuba", async function() {
+    let users = await getUsers(['iliakan', 'remy', 'brak.użytkowników']);
```

Comment:
Chyba lepiej nazwać to "nieistniejący.użytkownik", bo taki jest sens tego elementu.

Reply:
```suggestion
    let users = await getUsers(['iliakan', 'remy', 'nieistniejący.użytkownik']);
```

Reply:
Jasne.